docs/docusaurus/docs/guides/connecting_to_your_data/cloud/s3/components_pandas/_instantiate_your_projects_datacontext.mdx

```diff
@@ -1,10 +1,10 @@
 
-Import these necessary packages and modules.
+Run the following code to import the packages and modules:
 
 ```python name="tests/integration/docusaurus/connecting_to_your_data/cloud/s3/pandas/inferred_and_runtime_yaml_example.py imports"
```

Comment:
With FDS, I don't think we use anything here except:
```
import great_expectations as gx
```

Double check that the other imports and modules aren't used in the code examples for connecting to data (I didn't see them when I looked through) and if you don't see them either go ahead and edit the code for this tag to only include the `great_expectations` import.